### ISSUE TITLE
Size options for inputs

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#1880]
The idea is to have different size options. As sometimes larger inputs is preferrable (header search), and sometimes it's smaller ones (i.e. table header).

Do you have any plans on smaller/bigger inputs @xx13?

**Comment by xx13** [comment]
@asvae yep, it was in the backlog along with fixes for notes, errors etc. those sizes will be most likely synced with button sizes
